csrc/options.cpp

```diff
@@ -101,6 +101,7 @@ template <>
 std::unordered_map<DebugDumpOption, std::vector<std::string>> Options<
     DebugDumpOption>::getOptionsFromEnv() {
   const std::unordered_map<std::string, DebugDumpOption> available_options = {
+      {"!", DebugDumpOption::CallStack},
```

Comment:
How about a less ambiguous name like ftrace or something?

Reply:
changed to ftrace